torch_geometric/nn/conv/peg_conv.py

```diff
@@ -0,0 +1,149 @@
+import torch
+from torch import Tensor, nn
+from torch.nn import Parameter
+from torch_sparse import SparseTensor
+
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.nn.conv.gcn_conv import gcn_norm
+from torch_geometric.nn.dense.linear import Linear
+from torch_geometric.nn.inits import zeros
+from torch_geometric.typing import Adj, OptTensor, Tensor
+
+
+class PEGConv(MessagePassing):
+    r"""The PEG layer from the
+    `"Equivariant and Stable Positional Encoding
+    for More Powerful Graph Neural Networks"
+    <https://arxiv.org/abs/2203.00199>`_ paper.
+
+    $$X^{'},Z^{'}=(\sigma [(\hat{A} \odot M)XW],Z)$$
```

Comment:
Can we follow the `.. math::` syntax here as in other layers? The formula also looks wrong: https://pytorch-geometric--4571.org.readthedocs.build/en/4571/modules/nn.html#torch_geometric.nn.conv.PEGConv